source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -301,13 +297,9 @@ package:
    * - ``BsonProperty``
      - Specifies a custom document field name when converting the POJO
        field to BSON. You can include a discriminator to serialize POJOs
-       nested within the field.
-
-       .. important::
-
-          When applying ``@BsonProperty`` to a private field, you must also add
-          getter and setter methods for that field to serialize and customize
-          the field name.
+       nested within the field. When applying ``@BsonProperty`` to a private field, 
+       **you must also add getter and setter methods for that field to serialize and 
+       customize the field name**.
```

Comment:
```suggestion
       nested within the field. 
       
       When applying ``@BsonProperty`` to a private field, 
       **you must also add getter and setter methods for that field to serialize and 
       customize the field name**.
```
Suggestion: Separate implementation detail from rest of definition for ease of reading.